cypress/e2e/moderationDashboard.cy.ts

```diff
@@ -41,9 +41,9 @@ describe(
                     )
             });
 
-            it("it shows the moderation top nav", () => {
+            it.skip("it shows the moderation top nav", () => {
```

Comment:
Approving the PR, but I think we should remove the `.skip` before merging?

Reply:
since this one is failing unrelated to this PR, let's fix it separately. (I tried initially and there wasn't a quick fix)

Reply:
https://github.com/ourjapanlife/findadoc-web/issues/540 created!